app/controllers/api/mentor_discussion_posts_controller.rb

```diff
@@ -35,6 +35,20 @@ def create
       head 200
     end
 
+    def update
+      post = Solution::MentorDiscussionPost.find_by(uuid: params[:id])
+
+      return render_404(:mentor_discussion_post_not_found) if post.blank?
+      return render_403(:permission_denied) unless post.author == current_user
+
+      if post.update(content_markdown: params[:content])
+        DiscussionPostListChannel.notify!(post.discussion, post.iteration)
+        render json: SerializeMentorDiscussionPost.(post, current_user)
```

Comment:
Consider wrapping this in an object `{ post: ... }`, so you can change the metadata / more serializers later. Even if you don't / won't do that: checking `post` exists and then converting the type in typescript is safer than not checking anything at all.